axiom/nr_php_packages.c

```diff
@@ -56,6 +56,23 @@ void nr_php_packages_add_package(nr_php_packages_t** h, nr_php_package_t* p) {
     return;
   }
 
+  // If package with the same key already exists, we will check if the value is
+  // different. If it is different, then we will update the value of the package
+  if (0
+      != nr_php_packages_has_package(*h, p->package_name,
+                                     nr_strlen(p->package_name))) {
+    nr_php_package_t* package = (nr_php_package_t*)nr_hashmap_get(
+        *h, p->package_name, nr_strlen(p->package_name));
```

Comment:
Wouldn't it be more efficient to just call `nr_hashmap_get`? This way the `nr_hashmap_hash_key` gets calculated once and `nr_hashmap_fetch` is also called once.

Reply:
Great catch! I have removed the unnecessary function call [a486c77](https://github.com/newrelic/newrelic-php-agent/pull/754/commits/a486c774a35994aa1441b77d5bcf199659e0c8c4)